src/rules/expect-expect.ts

```diff
@@ -1,10 +1,35 @@
 import { Rule } from 'eslint';
 import * as ESTree from 'estree';
-import { isExpectCall, isTest } from '../utils/ast';
+import { isExpectCall, isIdentifier, isTest } from '../utils/ast';
+import { Settings } from '../utils/types';
+
+type Options = {
+  additionalAssertFunctionNames?: readonly string[];
+};
+
+function isAssertionCall(
+  node: ESTree.CallExpression,
+  additionalAssertFunctionNames: readonly string[]
+) {
```

Comment:
Lot's of `readonly` in this file, not really necessary

Reply:
It doesn't create a restriction to the passer, just consumer. I can remove it if you like, but it's useful type information for both perspectives (function writer knows/is disallowed from mutation, caller knows mutation won't happen), in my experience.

Reply:
It's unnecessarily verbose, so please remove it.